packages/augur-core/source/contracts/GnosisSafeRegistry.sol

```diff
@@ -0,0 +1,53 @@
+pragma solidity 0.5.10;
+
+import 'ROOT/Augur.sol';
+import 'ROOT/external/IGnosisSafe.sol';
+import 'ROOT/external/IProxyFactory.sol';
+
+
+contract GnosisSafeRegistry {
+    // mapping of user to created safes. The first safe wins but we store additional safes in case a user accidentally makes multiple.
+    mapping (address => IGnosisSafe[]) public accountSafes;
+
+    IAugur public augur;
+    bytes32 public proxyCodeHash;
+    address public gnosisSafeMasterCopy;
+
+    function initialize(IAugur _augur) public returns (bool) {
+        augur = _augur;
+        gnosisSafeMasterCopy = _augur.lookup("GnosisSafe");
+        IProxyFactory _proxyFactory = IProxyFactory(_augur.lookup("ProxyFactory"));
+        proxyCodeHash = keccak256(_proxyFactory.proxyRuntimeCode());
+        return true;
+    }
+
+    // The misdirection here is becuase this is called through a delegatecall execution initially. We just direct that into making an actual call to the register method
```

Comment:
```suggestion
    // The misdirection here is because this is called through a delegatecall execution initially. We just direct that into making an actual call to the register method
```